committee-code-of-conduct/election.md

```diff
@@ -68,19 +74,19 @@ lists, historical data, and any other activities related to the CoCC.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected committee
+In the event of a resignation or other loss of an elected CoCC
 member, the candidate with the next most votes from the previous election will
 be offered the seat.  This process will continue until the seat is filled.
 
 In case this fails to fill the seat, a special election for that position will
 be held as soon as possible. Eligible voters from the most recent election
 will vote in the special election (ie: eligibility will not be redetermined
-at the time of the special election). A committee member elected in a special
+at the time of the special election). A CoCC member elected in a special
 election will serve out the remainder of the term for the person they are
 replacing, regardless of the length of that remainder.
 
-In the event that the committee votes to dissolve in its entirety an entirely
-new special election will be held by the steering committee instead of filling
+In the event that the CoCC votes to dissolve in its entirety, an entirely
+new special election will he held by the Steering Committee instead of filling
 with names from the previous election. Those elected will complete the remaining
-terms of the previous, dissolved, committee. With those receiving the highest
+terms of the previous, dissolved CoCC's members. Those receiving the highest
```

Comment:
This sentence fragment reads a bit odd. Maybe “Candidates receiving the highest number of votes will receive the longest term from thise seats open for election.”?